src/room.py

```diff
@@ -1,2 +1,11 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+class Room:
+    def __init__(self, name, prompt, items):
```

Comment:
Very slick right here! If you wanted to have an empty array/no items for certain rooms when you create them you could also set a default empty array by simply saying -->  def __init__(self, name, prompt, items = [ ]).